README.md

```diff
@@ -1,2 +1,69 @@
 # identity-samples-dotnet
-Samples related to the identity written using .NET technologies.
+
+Samples related to the WSO2 Identity Server written using .NET technologies. Currently below listed Samples are 
+availbale in this repository
+
+1. OIDC SDK Sample (Login_APP)
```

Comment:
Rename it to,
Pickip-Manager

Under that have a small explanation on what it capable. ie, This sample is capable of demonstrating authentication with WSO2 Identity Server using the OIDC protocol. This is written using the .net-oidc-sdk.